### DISCUSSION TITLE
[Bug]: PDF Generation - Chromium use binary

### DISCUSSION THREAD

**Discussion by bellu** [pimcore/pimcore#17331] [Q&A / Support] [answered]
### Pimcore version

11.2.3

### Steps to reproduce

https://pimcore.com/docs/platform/Pimcore/Installation_and_Upgrade/System_Setup_and_Hosting/Additional_Tools_Installation/#chromium-chrome-headless---deprecated

It seems to be possibile to use the Chromium binary locally, but it doesn't specify where should i write the path of the installed  chromium.

### Actual Behavior

The only param that i can set for chromium in the config.yaml is the URI.

### Expected Behavior

I would like to set the source of the binary locally, like it's explained in the documentation.

**Comment by kingjia90** [comment, marked as answer]
If i got it right, you don't need to prrovide a path to the library, it needs to be installed in the server 
https://github.com/pimcore/pimcore/blob/4e3c0060d29c7117bbd681684c413ab4301d92fa/lib/Image/HtmlToImage.php#L82-L91
and be an executable program via terminal by typing `chromium` or `chrome`
